logstash-filter-syslog_pri.gemspec

```diff
@@ -22,6 +22,9 @@ Gem::Specification.new do |s|
   # Gem dependencies
   s.add_runtime_dependency "logstash-core", '>= 1.4.0', '< 2.0.0'
 
+  s.add_development_dependency 'logstash-filter-grok'
```

Comment:
@suyograo yeah! will change to handcrafted events. As I said I introduced grok to be sure the lines where syslog ones, but your point if valid too.